src/SimpleApi/ProductMigration.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using MongrationDotNet;
+
+namespace SimpleApi
+{
+    public class ProductMigration : CollectionMigration
+    {
+        public override Version Version => new Version(1, 1, 1, 0);
+        public override string Description => "Product migration";
+        public override bool MigrateArrayValues { get; } = true;
```

Comment:
I wonder if the migration where withMigrateArrayValues =true should be a new type CollectionArraysMigration : CollectionMigration

Reply:
I think otherwise. Users may want to rename 1 normal field and 1 array field of same collection and he/she has to create 2 migrations. Also by splitting the migration into two we are exposing that they are different whereas this way, users doesn't care about the implementation as they are both renaming with just the different type